PurchasesCoreSwift/Misc/SystemInfo.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  SystemInfo.swift
+//  PurchasesCoreSwift
+//
+//  Created by Joshua Liebowitz on 6/29/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+import UIKit
+#elseif os(watchOS)
+import UIKit
+import WatchKit
+#elseif os(macOS)
+import AppKit
+#endif
+
+public typealias RCSystemInfo = SystemInfo
```

Comment:
why is this needed?

Reply:
It no longer is. Since I was migrating so many files, the reference to this class needed RCSystemInfo for a minute.